examples/tests_ci/run_test_sink_synchro.py

```
@@ -206,5 +206,41 @@ def main():
 
     check_ref_dataset(model2.get_sinks())
 
+    dic = ctx2.collect_data()
+    assert 2266 == len(dic["xyz"])
```

Ruff (0.16.1)

[warning] Yoda condition detected

Rewrite as `len(dic["xyz"]) == 2266`

(SIM300)